assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -9,110 +10,118 @@ import { get } from '../../utils/routeMap';
-            <div className="footer-links">
-              <div className="copyright">
-                ©{' '}
-                <Link to="http://assembl.bluenove.com/" target="_blank">
-                  Assembl
-                </Link>{' '}
-                powered by{' '}
-                <Link to="http://bluenove.com/" target="_blank">
-                  bluenove
-                </Link>
-              </div>
-              <div className="terms">
-                {hasTermsAndConditions && (
-                  <div className="terms-of-use">
-                    <Link to={`${get('terms', slug)}`}>
-                      <Translate value="footer.terms" />
-                    </Link>
-                  </div>
-                )}
-                {hasLegalNotice && (
-                  <div className="legal-notice">
-                    {hasTermsAndConditions ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                    <Link to={`${get('legalNotice', slug)}`}>
-                      <Translate value="footer.legalNotice" />
-                    </Link>
-                  </div>
-                )}
-              </div>
-              <div className="terms">
-                {hasCookiesPolicy && (
-                  <div className="cookie-policy">
-                    <Link to={`${get('cookiesPolicy', slug)}`}>
-                      <Translate value="footer.cookiePolicy" />
-                    </Link>
-                  </div>
-                )}
-                {hasPrivacyPolicy && (
-                  <div className="privacy-policy">
-                    {hasCookiesPolicy ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                    <Link to={`${get('privacyPolicy', slug)}`}>
-                      <Translate value="footer.privacyPolicy" />
-                    </Link>
-                  </div>
-                )}
-                {hasUserGuidelines && (
-                  <div className="user-guidelines">
-                    {hasPrivacyPolicy || hasCookiesPolicy ? <span className="small-hyphen-padding"> &mdash; </span> : null}
-                    <Link to={`${get('userGuidelines', slug)}`}>
-                      <Translate value="footer.userGuidelines" />
-                    </Link>
-                  </div>
-                )}
-              </div>
-              {assemblVersion && <div className="assembl-version">v{assemblVersion}</div>}
             </div>
+          )}
+          {footerLinks && (
+            <div className="custom-links">
+              {footerLinks.map((footerLink, index) => (
+                <div className="inline margin-m" key={`fl-${index}`}>
+                  <Link to={footerLink.url} target="_blank">
+                    {footerLink.titleEntries[lang]}
+                  </Link>
+                </div>
+              ))}
+            </div>
+          )}
+          <div className="footer-links">
+            <div className="copyright">
+              ©{' '}
```

Comment:
eventually use :before and :after css selector to handle white space, instead of having {' '}